document/src/main/java/com/ritense/document/domain/Document.java

```diff
@@ -42,6 +43,15 @@ public interface Document {
     @JsonProperty
     DocumentDefinition.Id definitionId();
 
+    @JsonProperty
+    String assigneeEmail();
```

Comment:
Even though i don't mind having the email, i think we always also want the user ID

Reply:
change email to id